### ISSUE TITLE
Visualize geometry optimization trajectory and torsion scans

### ISSUE THREAD

**Issue by ChayaSt** [MolSSI/QCFractal#374]
**Is your feature request related to a problem? Please describe.**
When exploring a geometry optimization trajectory or a torsion scan, each molecule needs to be loaded separately which can become cumbersome. 

**Describe the solution you'd like**
Something like NGLViewer to scroll through conformations in a torsion scan or geometry optimization.

**Comment by dgasmith** [comment]
Yup, this is something that we have been looking at doing for some time. The major issue is that our  3dMol.js pick for molecular visualization is not DOM-wrapped which means that its widget integration is limited. I think we will have to switch over to nglview for this kind of tech, but nglview doesn't have the volumetric rending for orbitals... this will be fun.

This is something I will get back to early September unless someone else tackles it first.

**Comment by hainm** [comment]
> but nglview doesn't have the volumetric rending for orbitals.

@dgasmith does orbital's volume data has its own format? 
Those are supported ones in `NGL`: MRC/MAP/CCP4, DX/DXBIN, CUBE, BRIX/DSN6, XPLOR/CNS

**Comment by dgasmith** [comment]
@hainm We could give you any of those formats. Looking through your docs I see [this](https://github.com/arose/nglview/blob/master/examples/images/display_ccp4_file.md) which I missed before (sorry!). Could you provide orbital rendering similar to what you see in the bottom left [here](https://3dmol.csb.pitt.edu)?

**Comment by hainm** [comment]
Do you have sample file? Thanks.

**Comment by hainm** [comment]
> Do you have sample file? Thanks.

nevermind, it's in 3dmol repo.

**Comment by hainm** [comment]
> Could you provide orbital rendering similar to what you see in the bottom left here?

I think so, close enough, please see below:
![volume](https://user-images.githubusercontent.com/4451957/63457175-786d4200-c41e-11e9-8c23-d950aa20af54.png)

**Comment by dgasmith** [comment]
Ok, I'm sold :). I will look into integration when I get from a few trips.

PS: Will you happen to be at ACS? Might be a good time to hack a few things out.

**Comment by hainm** [comment]
> PS: Will you happen to be at ACS?

No I won't. Do feel free do open any issue in nglview.
